src/vicsetup/libvicsetup/argon2_argon2.c

```diff
@@ -0,0 +1,3 @@
+#include "argon2.h"
```

Comment:
Why are all those files needed? Looks like a weird work-around/short-cut for something?

Reply:
argon2 does not compile cleanly so this was to workaround various warnings (vicsetup treats warnings as errors). I will extend the build script to handle this on a per-file basis and eliminate these files.

Reply:
Also, this permitted us to override **malloc/free** with **vic_malloc/vic_free** but this appears to no longer be necessary so I will remove it.

Reply:
Removed custom vic allocator and then argon2 wrapper files.